lms/djangoapps/api_manager/users/views.py

```diff
@@ -143,7 +143,10 @@ def _manage_role(course_descriptor, user, role, action):
     Helper method for managing course/forum roles
     """
     supported_roles = ('instructor', 'staff', 'observer', 'assistant')
-    forum_moderator_roles = ('instructor', 'staff', 'assistant')
+    role_perm_map = {
+        FORUM_ROLE_MODERATOR: ('instructor', 'staff'),
+        FORUM_ROLE_COMMUNITY_TA: ('assistant',),
```

Comment:
@Kelketek Wouldn't it be better to invert this dictionary?

```python
role_perm_map = {
    'staff': FORUM_ROLE_MODERATOR,
    ...
}
```

Then you wouldn't need the `for` loops below; you could just use key access.

Also, can you use the constants like `CourseStaffRole.ROLE` instead of strings?